packages/react/test/specs/components/Attachment/Attachment-test.tsx

```diff
@@ -18,5 +21,30 @@ describe('Attachment', () => {
     handlesAccessibility(Attachment, {
       defaultRootRole: undefined,
     })
+
+    test('handleClick is executed when Enter is pressed for attachment, not for more option button', () => {
```

Comment:
Can we separate these in two tests (one when it should be clicked and one where it should not be clicked)?